### PR TITLE
Demo of Using VisualDL in PyTorch

### PR DESCRIPTION
**Pull request by daming-lu**

### CODE REVIEW

File: demo/pytorch/TUTORIAL_CN.md [hunk 1]
```diff
@@ -0,0 +1,211 @@
+# 如何在PyTorch中使用VisualDL
+
+下面我们演示一下如何在PyTorch中使用VisualDL，从而可以把PyTorch的训练过程以及最后的模型可视化出来。我们将以PyTorch用卷积神经网络(CNN, Convolutional Neural Network)来训练
+[Cifar10](https://www.cs.toronto.edu/~kriz/cifar.html) 数据集作为例子。
+数据集作为例子。
```
Comment (jetfuel) [inline demo/pytorch/TUTORIAL_CN.md]: extra 数据集作为例子

Comment (jetfuel) [inline demo/pytorch/TUTORIAL_CN.md]: extra 数据集作为例子

File: demo/pytorch/TUTORIAL_CN.md [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+        running_loss += loss.data[0]
+        if i % 2000 == 1999:    # print every 2000 mini-batches
+            print('[%d, %5d] loss: %.3f' %
+                  (epoch + 1, i + 1, running_loss / 2000))
+            running_loss = 0.0
+
+print('Finished Training')
+```
+
+最后，因为 PyTorch 采用 Dynamic Computation Graphs，我们用一个 dummy 输入来空跑一下模型，以便产生图
```
Comment (jetfuel) [inline demo/pytorch/TUTORIAL_CN.md:172]: I don't quite understand this sentence. Why does Dynamic Computation graphs need dummy?

**Review by jetfuel — DISMISSED** [review]
Minor changes, otherwise LGTM

File: demo/pytorch/TUTORIAL_CN.md (hunk 2 above)
Comment (daming-lu) [reply demo/pytorch/TUTORIAL_CN.md:172]: "PyTorch is even more flexible and uses something known as dynamic computation graphs, in which the computational graph is generated and executed on the fly. "

File: demo/pytorch/TUTORIAL_CN.md (hunk 1 above)
Comment (daming-lu) [reply demo/pytorch/TUTORIAL_CN.md]: done

**Review by Superjomn — APPROVED** [review]
LGTM

**Pull request merged**